### DISCUSSION TITLE
Only one instance(tab) of Storybook will load

### DISCUSSION THREAD

**Discussion by joshwooding** [storybookjs/storybook#16446] [Help] [answered]
When using `storyStoreV7` opening the same Storybook instance in different tabs only the first one loads completely. As soon as you close the first tab the next one opened will load. Is this intended behaviour?

System Info:
Windows

**Comment by shilman** [comment]
That sounds like a bug to me. @tmeasday did you happen to test this out?

**Reply by tmeasday** [reply]
How bizzarre. Yes I am seeing similar behaviour. I have no clue why it might be happening 🤔

**Reply by tmeasday** [reply]
Hmm, OK so it is a browser limit around max open requests, due to the always open SSE-events for `stories.json`.

Apparently if we switched to a websocket, we wouldn't have this issue. Kind of a frustrating limitation for localhost.

**Comment by joshwooding** [comment, marked as answer]
This should be solved in https://github.com/storybookjs/storybook/pull/16504
